### ISSUE TITLE
H5P container not showing content at full width.

### ISSUE THREAD

**Issue by educationfutures** [EvokeNet/moodle-mod_evokeportfolio#23]

**Comment by educationfutures** [comment]
![Screen Shot 2021-08-20 at 8 12 56 AM](https://user-images.githubusercontent.com/51122406/130238735-bb41b30d-cc3b-46aa-891f-4c29ae9e35cc.png)

**Comment by willianmano** [comment]
@educationfutures this area is responsible to display all types of activities, including H5P. What do I need to do? Do not limit the main area width, and enable H5P only, it to be displayed on the entire page?

**Comment by educationfutures** [comment]
Yeah. Let’s check with Martine and Jorge what width it should be displayed as and where the container should be placed on the screen (does it need to be centered on desktop/lXL width?)

**Comment by willianmano** [comment]
The content was centered in the first version, but in the last Martine adjustments, she requested to align it to the left.

**Comment by educationfutures** [comment]
Okie-dokie. I didn’t realize that. Let’s leave this issue open for the time being and bounce it by Martine for a second opinion.
